dev/com.ibm.ws.microprofile.faulttolerance.cdi_fat/fat/src/com/ibm/websphere/microprofile/faulttolerance_fat/suite/RepeatFaultTolerance.java

```diff
@@ -69,7 +69,7 @@ public static FeatureReplacementAction ft20Features(String server) {
      */
     public static RepeatTests repeatDefault(String server) {
         return RepeatTests.with(mp20Features(server))
-                        .andWith(ft20Features(server));
+                        .andWith(ft20Features(server).fullFATOnly());
```

Comment:
Can we keep ft20Features in lite and move mp20Features to full instead?

FT 2.0 is the current version, mp20Features includes FT 1.1.

Also, please update the comment to point out that FT 1.1 is only for FULL mode